app/controllers/users_controller.rb

```diff
@@ -14,6 +14,11 @@ def create
     end
   end
 
+  def email_taken
```

Comment:
Is this endpointed authed? It shouldn't be. And can it just be a get with a query param?